Working-Sessions/Threat-Model/TM-What-can-go-wrong-with-JuiceShop.md

```diff
@@ -6,7 +6,7 @@ track        : Threat Model
 technology   :
 related-to   : Juice Shop
 when-day     : Tue
-when-time    : PM-1
+when-time    : eve-3
 location     : Room-6
```

Comment:
Hi @adam234 

I believe the mapping you want is

when-day     : Wed
when-time    : Eve

eve-3 is not supported by the schedule engine

Can you update this with a fix? (if you push into the same branch it will appear here automatically, i.e. no need to create another Pull Request)